test/ext/ws.js

```diff
@@ -88,12 +89,49 @@ describe("web-sockets extension", function () {
         this.messages.length.should.equal(1);
     })
 
-    it('handles message from the server', function () {
+    it('handles message from the server (text)', function () {
         var div = make('<div hx-ext="ws" ws-connect="ws://localhost:8080"><div id="d1">div1</div><div id="d2">div2</div></div>');
         this.tickMock();
 
         this.socketServer.emit('message', "<div id=\"d1\">replaced</div>");
 
+        setTimeout(function() {
+            byId("d1").innerHTML.should.equal("replaced");
+            byId("d2").innerHTML.should.equal("div2");
+        }, 10);
+    })
+    
```

Comment:
This `setTimeout` call also won't work as expected


```suggestion
        this.tickMock()

        byId("d1").innerHTML.should.equal("replaced");
        byId("d2").innerHTML.should.equal("div2");
    })

```